build.rs

```diff
@@ -32,7 +33,16 @@ fn main() {
         .unwrap()
         .read_to_string(&mut contents)
         .unwrap();
-    let new_contents = format!("mod proto {{pub mod {} {{ \n{}\n}}}}", name, contents);
+
+    // Work around for https://github.com/rust-lang/rust/issues/5472
+    // Introduced in rust-protobuf https://github.com/stepancheg/rust-protobuf/pull/495
+    // More discussion: https://github.com/stepancheg/rust-protobuf/pull/523#issuecomment-701026992
+    let filtered: String = contents
+        .lines()
+        .filter(|line| line.trim() != "#![rustfmt::skip]")
+        .join("\n");
```

Comment:
This maybe feels a little over-engineered with the introduction of itertools as a build dependency (though I admit I haven't fully thought through whether it's negative to include it since we already have it as a regular dependency).
Would it be any worse to just do `let filtered: String = contents.replace("#![rustfmt::skip]", "");`

Reply:
It's certainly not the same behavior, though I can't tell you if the source could legally contain the code that would actually make it break. 

I like the itertools solution much better since it's more clear what will happen to each line exactly. 

I hope that rust will only build itertools once since it's matching the same version.

Reply:
I'm not too concerned about bringing in itertools at build time since I think we use it in nearly every rust codebase. It's like `std+` to me. Also, we either need to get off of rust-protobuf (prost) or figure out a better way of doing things longterm. In the short-term we need to get to 2.17 so we can use the "well-known-types" that google puts out for protobuf3.

Reply:
I realize I didn't really address your concern in my last comment, @giarc3. Sorry. 

I think processing line by line is safer. There are certainly other ways of doing it, but I don't think it's worth fretting over it too much for this workaround.